<launch>
  <arg name="robot_label" value="$(optenv TURTLEBOT_LABEL robot_0)" />
  <group ns="$(arg robot_label)">
    <param name="tf_prefix" value="$(arg robot_label)" />
    <node pkg="custom_navigation" type="adcDriver" name="adcDriver" respawn="false" />
    <node pkg="custom_navigation" type="adcParams" name="adcParams" respawn="false" >
      <rosparam file="$(find custom_navigation)/param/gyro/$(arg robot_label).yaml" command="load"/>
    </node>
    <node pkg="custom_navigation" type="odom_gyro" name="odom_gyro" respawn="false" output="screen"/>  
    <!--  fake sensor for amcl -->
    <node pkg="custom_navigation" type="robot_obstacle" name="robot_obstacle" respawn="false" >
      <param name="robot_name" value="$(arg robot_label)" />
    </node>
</group>


  
  <!--  fake positions for amcl simulation 
  <node pkg="custom_navigation" type="robot_position" name="$(anon robot_position)" respawn="false" >
    <param name="robot_name" value="robot_2" />
  </node>
  <node pkg="custom_navigation" type="robot_position" name="$(anon robot_position)" respawn="false" >
    <param name="robot_name" value="robot_3" />
  </node>  
  <node pkg="custom_navigation" type="robot_position" name="$(anon robot_position)" respawn="false" >
    <param name="robot_name" value="robot_4" />
  </node>  
  <node pkg="custom_navigation" type="robot_position" name="$(anon robot_position)" respawn="false" >
    <param name="robot_name" value="robot_5" />
  </node>-->
  
</launch>
